c7n/resources/rds.py

```diff
@@ -233,6 +233,16 @@ def process(self, dbs, event=None):
         return self.get_matching_aliases(dbs)
 
 
+@filters.register('age')
+class RdsInstanceAge(AgeFilter):
+
+    schema = type_schema(
+        'age',
+        days={'type': 'number'},
+        op={'type': 'string', 'enum': OPERATORS.keys()})
+    date_attribute = 'InstanceCreateTime'
```

Comment:
we've been moving away from custom age filters per resource. The generic value filter supports conversion of any attribute into a datetime for this comparison. ie. value_type: age  key: InstanceCreateTime is equivalent.